packages/fluentui/react-northstar/src/themes/teams/components/Button/buttonStyles.ts

```diff
@@ -211,6 +211,10 @@ const buttonStyles: ComponentSlotStylesPrepared<ButtonStylesProps, ButtonVariabl
           ...getIconFillOrOutlineStyles({ outline: false }),
           color: v.textColorIconOnlyHover,
```

Comment:
Wonder if these should be removed instead since your fix is to re-enforce the inverted background & foreground colors...

Reply:
I inverted the condition to not re-enforce the inverted background & foreground colors.

Reply:
@notandrew should I remove the addition of the inverted iconOnly button in the docs or should it stay and the screener tests updated?

Reply:
I'd update the screener tests - I think that's the right way to go. :)